lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -4087,32 +4114,64 @@ void request_variables(const llvm::json::Object &request) {
 void request_locations(const llvm::json::Object &request) {
   llvm::json::Object response;
   FillResponse(request, response);
-  auto arguments = request.getObject("arguments");
+  auto *arguments = request.getObject("arguments");
```

Comment:
please remove the auto here. An explicit type would make this more readable

Reply:
Note that all other request handlers also use `auto*`.
I only added the `*` here for consistency with other similar functions